eth/backend.go

```diff
@@ -234,7 +234,10 @@ func New(ctx *node.ServiceContext, config *Config) (*Ethereum, error) {
 	}
 	eth.APIBackend.gpo = gasprice.NewOracle(eth.APIBackend, gpoParams)
 
-	eth.APIBackend.l1gpo = gasprice.NewL1Oracle(config.Rollup.L1GasPrice)
+	// create the L1 GPO and allow the API backend and the sync service to access it
+	l1Gpo := gasprice.NewL1Oracle(config.Rollup.L1GasPrice)
+	eth.APIBackend.l1gpo = l1Gpo
```

Comment:
You could deduplicate needing the `l1gpo` on the `APIBackend` because the `SyncService` is accessible within it, see: https://github.com/ethereum-optimism/go-ethereum/blob/2dd129ff7b49c0c324844d8b18b41200c2f1a81e/eth/api_backend.go#L67-L69

Reply:
I'd rather keep it as-is, to avoid having to expose new getter methods, or make the gpo public. Do you strongly disagree with the way it is now?

Reply:
Not super opinionated on it, just thought it could save a little code duplication but your point is fine